webrtc.html

```diff
@@ -1417,6 +1417,17 @@ <h4>Set the RTCSessionDescription</h4>
                   </li>
                 </ol>
               </li>
+              <li>
+                <p>If <var>description</var> is set as a remote description and
```

Comment:
It doesn't make sense to apply the description before you have verified the identity, I think.

Reply:
@alvestrand Good point. @martinthomson WDYT?